modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -347,6 +350,13 @@ def width(self, materialize=True):
             self._width_cache = self.apply(self._width_extraction_fn()).get()
         return self._width_cache
 
+    @property
+    def _identity(self):  # noqa: RT01
```

Comment:
Let's just add a `Returns` section and remove this noqa.

Reply:
Added